daffodil-core/src/main/scala/org/apache/daffodil/lib/iapi/Diagnostic.scala

```diff
@@ -128,8 +128,6 @@ abstract class Diagnostic protected (
    */
   override def getMessage(): String = message
 
-  override def getMessageOnly: String = message
```

Comment:
I recommend leaving this in.  First the name is clearer about what it is returning. Second, why make people change it again?

Reply:
I personally like the idea of getMessage returning only the message, and getMessageVerbose returning message+mode+context. I too got confused about why getMessage returned all those things

Reply:
+1 to the idea of a "getMessageVerbose" or equivalent function.

Reply:
toString already contains this:

> Get a string containing the mode name, message, schema location, and data location combined into a single string

I also just pushed a fixup to clarify getMessage and recommend toString so it now says:

>  Get the diagnostic mode name
>
> This does not include any additional details about the Diagnostic, such as
> mode name, schema context, or data location. For a human-readable
> representation of this diagnostic, including the message and all context
> information, use {@link #toString()}.

Reply:
Ah, I was looking at the toString definition in the Scala file, not the Java one.  The one in the Scala file doesn't have those details, so that's why I was confused.